superset-frontend/src/components/ErrorMessage/types.ts

```diff
@@ -88,7 +89,7 @@ export type ErrorType = ValueOf<typeof ErrorTypeEnum>;
 // Keep in sync with superset/views/errors.py
 export type ErrorLevel = 'info' | 'warning' | 'error';
 
-export type ErrorSource = 'dashboard' | 'explore' | 'sqllab';
+export type ErrorSource = 'dashboard' | 'explore' | 'sqllab' | 'crud';
```

Comment:
`crud` doesn't seem like a domain source like others here. Is there a better name for this?

Reply:
Yeah, I couldn't think of a better name. The CRUD is one of the main domains, just not a fancy one. :-P